app/models/xccdf/benchmark.rb

```diff
@@ -43,6 +49,15 @@ def find_latest(benchmarks)
           Gem::Version.new(benchmark.version)
         end
       end
+
+      def latest_supported
+        Rails.cache.fetch('latest_supported_benchmarks') do
+          where(
+            ref_id: LATEST_SUPPORTED_VERSIONS.keys,
+            version: LATEST_SUPPORTED_VERSIONS.values
+          ).compact
```

Comment:
This doesn't do what you want:

```sql
SELECT "benchmarks".* FROM "benchmarks" WHERE "benchmarks"."ref_id" IN ('xccdf_org.ssgproject.content_benchmark_RHEL-6', 'xccdf_org.ssgproject.content_benchmark_RHEL-7', 'xccdf_org.ssgproject.content_benchmark_RHEL-8') AND "benchmarks"."version" IN ('0.1.28', '0.1.46', '0.1.46')
```

You've got to use something like this `inject` construct:

```rb
pry(main)> puts LATEST_SUPPORTED_VERSIONS.inject(Xccdf::Benchmark.where('1=0')) { |rel, arr| rel.or(Xccdf::Benchmark.where(ref_id: arr[0], version: arr[1])) }.to_sql
SELECT "benchmarks".* FROM "benchmarks" WHERE ((((1=0) OR "benchmarks"."ref_id" = 'xccdf_org.ssgproject.content_benchmark_RHEL-6' AND "benchmarks"."version" = '0.1.28') OR "benchmarks"."ref_id" = 'xccdf_org.ssgproject.content_benchmark_RHEL-7' AND "benchmarks"."version" = '0.1.46') OR "benchmarks"."ref_id" = 'xccdf_org.ssgproject.content_benchmark_RHEL-8' AND "benchmarks"."version" = '0.1.46')
```